django/test/html.py

```diff
@@ -96,6 +97,19 @@ def _count(self, element, count=True):
                     elif element in child:
                         return 1
             else:
+                # Look for a complete sequence of element's children.
+                # Refs #27906.
```

Comment:
I think we could do with a better comment here (and/or in the block).

* Why do we need to check this case? 
* We're relying on `__eq__`
* But also there's the `# complete match`
* When we exit the `if` block, how have we progressed the search? 

It's not easy to see what's going on.